### ISSUE TITLE
[Feature Request] Data class for easy serialization

### ISSUE THREAD

**Issue by BobbyESP** [Kyant0/taglib#1]
It would be nice to have a data class for being able to transport data easily inside the app that this lib is being used, like in your before `Tag` library. Thanks!

**Comment by BobbyESP** [comment]
(I have made it myself, if you want it tell me!)

**Comment by Kyant0** [comment]
Do you want to add Serializable annotation for each data class?

**Comment by BobbyESP** [comment]
Sorry for being unclear before. I think that is a better way to create a data class like "MetadataProperties" that would be serializable. In its companion object, a converter from the already existent type alias "PropertyMap" to this new data class would be awesome

**Comment by BobbyESP** [comment]
Yes

**Comment by Kyant0** [comment]
These properties are various and numerous, using a pre-defined data class may lead to insufficiency or mass. You can weigh over The fields in your own data class according to your needs, which is detailed and brief appropriately.

**Comment by BobbyESP** [comment]
> These properties are various and numerous, using a pre-defined data class may lead to insufficiency or mass. You can weigh over The fields in your own data class according to your needs, which is detailed and brief appropriately.

True. Thanks anyways for the fast response! I've made myself one with the fields I need.

**Comment by Kyant0** [comment]
Leave it open until relevant sample code is added.

**Comment by BobbyESP** [comment]
I think that, indeed, this is not viable although it would simplify some things like in my case that I have to copy all the properties, modify them and after that, write the new fields in the file.
